api/Trace/PropagationSetter.php

```diff
@@ -14,5 +14,5 @@ interface PropagationSetter
      * @param string $value
      * @return void
      */
-    public function set($carrier, string $key, string $value) : void;
+    public function set(&$carrier, string $key, string $value) : void;
```

Comment:
You may want to update the docbloc for this one:
 
`* @param mixed  $carrier` 

and should probably read 

`* @param mixed  &$carrier`

It might help to make others recognize that this is pass by reference.

Reply:
Good catch! All fixed 👍🏾

Reply:
It looks like the `phpstan` tests fail when it is labeled as a reference parameter in the doc block. This [issue](https://github.com/phpstan/phpstan/issues/699) might provide some insight here. I'm not too sure if there's a way around this so it might be better to switch it back?

Reply:
Yeah swapping it back is totally fine with me.  Sorry for the goose chase 😿

Reply:
Swapped it back 👍🏾